src/dex/maker-psm/maker-psm.ts

```diff
@@ -653,8 +653,9 @@ export class MakerPsm
       // not used on the contract, but used for analytics
       destToken,
       fromAmount,
-      // as there's no slippage, use instead of toAmount
-      quotedAmount,
+      // // as there's no slippage, use instead of toAmount
+      // quotedAmount,
+      toAmount,
```

Comment:
@JackieJoo Could you please explain why we need this fix within LitePSM DEX integration?